components/admin/lessons/AdminLessonCard.tsx

```diff
@@ -1,11 +1,28 @@
 import React from 'react'
 import Link from 'next/link'
 import styles from '../../../scss/adminLessonCard.module.scss'
-import { Lesson } from '../../../graphql/'
+
+type Lesson = {
+  id: number
+  title: string
+  description: string
+  docUrl?: string | null
+  githubUrl?: string | null
+  videoUrl?: string | null
+  order: number
+  slug: string
+  chatUrl?: string | null
+  challenges: Array<{
+    id: number
+    title: string
+    description: string
+    order: number
+  }>
+}
 
 type Props = {
   lesson: Lesson
-  pendingFlaggedQuestions: number
+  pendingFlaggedQuestions: number | string
```

Comment:
Why is it number or string?

Reply:
So when there are no questions, it displays 'No'. I can changed it to just say `0` if that is what's preferred, then we can just leave it as a number type

Reply:
Yeah lets do that. For this specific context, It seems weird to me that this field can be either be number or string. Lets simplify things and change it so that it can only be number

Reply:
If you want `No` to be displayed when there are no pending questions, then do this on line 40:
```
{pendingFlaggedQuestions || 'No'} Pending Questions
```

This will reduce complexity because we are keeping the type as `number`. Future developers who touch this code won't have to think about whether to pass a number or string to `AdminLessonCard` -- they'll just simply pass a number.

Reply:
I ended up going with just showing 0.